pages/linux/edquota.md

```diff
@@ -0,0 +1,28 @@
+# edquota
+
+> Edit quota for a user or group. By default it operates on all file systems with quotas.
+> Quota information is stored permanently in the quota.user and quota.group files in the root of the file system.
+
+- Edit quota of current user (default):
+
+`edquota --user {{user}}`
```

Comment:
I'd rather remove it in both (if repquota also allows it being omitted), since `edquota --user myusername` seems rather wasteful. That way, we could get rid of the "(default)" in the description. What do you think?